One test failure that looks somewhat unrelated to me:
```
java.lang.AssertionError: Protocol(s) are not enabled: JNLP-connect,CLI-connect. Reason: Legacy Non-encrypted JNLP/CLI protocols should be enabled. Enabled protocols: CLI2-connect,JNLP2-connect,JNLP3-connect,JNLP4-connect,Ping
	at org.junit.Assert.fail(Assert.java:88)
	at jenkins.AgentProtocolTest.assertProtocols(AgentProtocolTest.java:127)
	at jenkins.AgentProtocolTest.assertProtocols(AgentProtocolTest.java:105)
	at jenkins.AgentProtocolTest.testShouldNotDisableProtocolsForMigratedInstances(AgentProtocolTest.java:69)
```

--- a/war/pom.xml
+++ b/war/pom.xml
@@ -134,7 +134,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.modules</groupId>
       <artifactId>sshd</artifactId>
-      <version>2.0</version>
+      <version>2.1</version>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.ui</groupId>

